package.json

```diff
@@ -41,9 +47,21 @@
       "types": "./_types/index.d.ts"
     }
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,json,md,ts,}": "prettier --write"
+  },
   "module": "./_esm/index.js",
   "prettier": {
-    "semi": false
+    "arrowParens": "avoid",
```

Comment:
I'm lately leaning towards letting `arrowParens` to its default `always`. Makes the code look much more consistent and may help with adding types, etc.